node/core/pvf/src/error.rs

```diff
@@ -43,29 +81,10 @@ pub enum PrepareError {
 	RenameTmpFileErr(String),
 }
 
-impl PrepareError {
-	/// Returns whether this is a deterministic error, i.e. one that should trigger reliably. Those
-	/// errors depend on the PVF itself and the sc-executor/wasmtime logic.
-	///
-	/// Non-deterministic errors can happen spuriously. Typically, they occur due to resource
-	/// starvation, e.g. under heavy load or memory pressure. Those errors are typically transient
-	/// but may persist e.g. if the node is run by overwhelmingly underpowered machine.
-	pub fn is_deterministic(&self) -> bool {
```

Comment:
Why did you remove this?

Reply:
I figured that now we can just match on the `Deterministic` variant directly, so this function seemed extraneous.

Reply:
Yeah for sure, but calling this method is less code 🤣 But yeah, no strong argument here.